scripts/release.sh

```diff
@@ -136,6 +136,41 @@ function setup_branch() {
   git fetch ${KNATIVE_UPSTREAM} ${RELEASE_BRANCH}:upstream/${RELEASE_BRANCH}
 }
 
+# Setup version, branch and release notes for a auto release.
+function prepare_auto_release() {
+  echo "Auto release requested"
+  TAG_RELEASE=1
+  PUBLISH_RELEASE=1
+
+  local tags="$(git tag | cut -d 'v' -f2 | cut -d '.' -f1-2 | sort | uniq )"
+  local branches="$( { (git branch -r | grep origin/release-) ; (git branch  | grep release-); } | cut -d '-' -f2 | sort | uniq)"
+  local last_version=""
+
+  for i in $branches; do
+    RELEASE_VERSION=$i
+    for j in $tags; do
+      last_version="v${j}.0"
+      if [[ "$i" == "$j" ]]; then
+        RELEASE_VERSION=""
+      fi
+    done
+  done
+
+  if [ "$RELEASE_VERSION" == "" ];then
+    echo "Everything is released. Nothing to do!"
```

Comment:
It's easy to miss this message in the log. I suggest being more verbose and also using a prefix like in lines 205-207.